### ISSUE TITLE
packages.sury.org Debian repository GPG key has been changed

### ISSUE THREAD

**Issue by oerdnj** [oerdnj/deb.sury.org#1129]
See https://www.patreon.com/posts/25451165 for more information.

This affects only Debian repositories hosted at packages.sury.org, not Ubuntu PPAs.

Please **READ** the post before asking questions here.

**Comment by todeveni** [comment]
Submitted key to https://keyserver.ubuntu.com, which we use as a central key store in our scripting.

**Comment by TheRojam** [comment]
No results found
No results found: No keys found

**Comment by todeveni** [comment]
> If I search with "B188E2B695BD4743"

Try adding `0x` in front, which implies searching by key hex id

**Comment by oerdnj** [comment]
@TheRojam **Where** do you search for it?

**Comment by oerdnj** [comment]
But generally, using full fingerprint should save you from the troubles of adding or not-adding `0x`.

**Comment by santicalvo** [comment]
I just updated debian repos and got a pubkey issue.

What should I do? How to add the Pub key?
Thanks

**Comment by oerdnj** [comment]
How about reading the aforementioned Patreon post?

**Comment by santicalvo** [comment]
> How about reading the aforementioned Patreon post?

Sorry, I just noticed :-) Thank you very much.

**Comment by kartagis** [comment]
Err, let me ask something. Why wasn't a key imported in the first place? àpt-apt-repository` said it did. If it has been imported, am I looking at the right place? i.e. /etc/apt/trusted.gpg.d/`. Because it wasn't there.

**Comment by oerdnj** [comment]
Sorry, but I don’t understand what you are asking? Could you rephrase your questions and be more descriptive?

**Comment by kartagis** [comment]
When I added the PPA, it told me that a key has been imported, but I can't still install anything without `--allow-unauthenticated`.

**Comment by TheRojam** [comment]
> @TheRojam **Where** do you search for it?
keys.gnupg.net
But i came along this problem, after reading the patreon post.

**Comment by oerdnj** [comment]
@kartagis This post is not about PPA, but about the Debian packages available from packages.sury.org - if you are adding it using `add-apt-repository` then you are doing it wrong.

**Comment by kartagis** [comment]
I was following [this post.](https://www.rosehosting.com/blog/how-to-install-php-7-3-on-ubuntu-16-04/)

**Comment by oerdnj** [comment]
This is not a chat room, if you need user support, please use AskUbuntu or such.  I am sorry, but I don't have time for user support.

I am also going to delete all comments that just hijacks this issue for non-related stuff in a moment.
